doc/guides/writing-tests.md

```diff
@@ -18,6 +18,13 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Where does the test go
+One can refer [test directory structure](https://github.com/nodejs/node/tree/master/test)
+to decide where to put your test cases. If you want to find any current test that exist then
+go to the defined [test directory structure](https://github.com/nodejs/node/tree/master/test) 
+and search for similar APIs and see if there is similar file available 
```

Comment:
A nit: missing period.

Reply:
did not get this one. Can you please elaborate

Reply:
Sorry. I think `file available` should be `file available.` (with a dot).

Reply:
resolved :)